### ISSUE TITLE
Context drawer margins are inconsistent

### ISSUE THREAD

**Issue by KoreshKrasa** [pop-os/libcosmic#755]
![screenshot-2024-12-14-17-26-38](https://github.com/user-attachments/assets/8ed5d8e4-9268-47e2-8416-e552113f9856)
Context drawer has margins on every edge except top. This can look a bit junky. Margin should be applied to all sides or none

**Comment by git-f0x** [comment]
It doesn't have padding at the top because a header bar is usually there. This also applies to the NavBar.
It might be easily fixable, be conditionally adding padding at the top when a window doesn't have a header bar.

**Comment by git-f0x** [comment]
Yeah, this is specific to cosmic-term, but I haven't yet found the place where the padding for the context drawer is set.
